Header header

geometry_msgs/Point center
float64[9] center_covariance

geometry_msgs/Vector3 axis
float64[9] axis_covariance

float64 theta
float64 theta_std_dev